Issue #642 - Expose config and INFO response for slave_priority

rave-eserating opened the pull request:
PR for enhancement: https://github.com/tidwall/tile38/issues/642

Allow setting slave_priority in the config to any integer value greater than or equal to zero.
Zero means the slave will never be promoted to master(leader), and any other positive numeric value will be used as the priority order for Sentinel slave election when the master goes down.

If not specified the default priority of a slave is 100. If a negative number is specified then slave_priority will not be output in the INFO response and that slave will be considered to have the default priority value of 100.

tidwall commented:
Hi @rave-eserating,

When testing on my side:

```
127.0.0.1:9851> config set slave_priority 10
(error) ERR Unsupported CONFIG parameter: slave_priority
```

Is this configuration intended to be only set from the config file?

rave-eserating commented:
@tidwall I have changed the config property name to be 'replica-priority' and I also added the code so that you can set/get via the config command. I tested both commands and they work as expected.

tidwall commented:
Looks good!
